src/Select.js

```diff
@@ -1,22 +1,28 @@
 import React from 'react';
 import styled from 'styled-components';
 import { Title } from './Title';
+import { FlexProperty } from './FlexProperty';
 
 const StyledSelect = styled.select`
-  border: 2px solid #333;
-  width: 100%;
   border-radius: 0;
+  border: 1px solid #aaa;
+  color: #333;
+  font-family: 'Roboto', sans-serif;
+  font-size: 17px;
+  height: 35px;
+  padding: 10px;
```

Comment:
I'd remove the padding here. What I understand from this code is that you're using `padding` to separate the `<Select />` component from the container.